ceremonies/src/lib.rs

```diff
@@ -266,7 +272,7 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			cid: CommunityIdentifier,
 			number_of_participants_vote: u32,
-			attestations: Vec<T::AccountId>,
+			attestations: BoundedVec<T::AccountId, T::MaxAttestations>,
```

Comment:
This could also just be `&[T::AccountId]` I think

```suggestion
			attestations: &[T::AccountId],
```

Reply:
f5b8a8af5efa494f0bbd43e059785f94b4eebadd

Reply:
Ah here, you actually can't because it is a top-level dispatchable.